src/convertToObject.js

```diff
@@ -3,16 +3,30 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles
+ * (see an example in [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related CSS properties
+ * (see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const result = {};
+  const cssArr = sourceString
+    .split(';')
+    .map(property => property.trim())
+    .map(property => property.split(/\s*:\s*/));
+
+  for (const property of cssArr) {
+    if (property.length === 2) {
+      result[property[0]] = property[1];
+    }
```

Comment:
- it's better to use destructuring here, as it will make a code cleaner 
- please also move 2 to the constant

```suggestion
    if (property.length === 2) {
      const [key, value] = property;
      result[key] = value;
    }
```